src/networks.c

```diff
@@ -656,10 +574,9 @@ void deleteNetwork(network_type *network) {
        deleteVector(network->arcs[i].classToll);
    }
 
-   deleteMatrix(network->demand, network->batchSize);
+   deleteMatrix(network->demand, network->numZones);

```

Comment:
using network->batchSize makes it more general to even batching the tntp networks. We changed this previously